google/cloud/firestore_v1/collection.py

```diff
@@ -272,6 +271,24 @@ def limit(self, count):
         query = query_mod.Query(self)
         return query.limit(count)
 
+    def limit_to_last(self, count):
```

Comment:
is adding this to collection required? does this match other language implementations better?

Reply:
There is a [test](https://github.com/googleapis/python-firestore/blob/b1c5987c606a14874b412e70f93015e161e278d6/tests/unit/v1/test_collection.py#L41-L49) which checks that `Query()` has the same (or less) methods that `CollectionReference()` does. So, as I understood that, the list of the supported methods should be the same for `Query()` and `CollectionReference()`, at least in our case.